### PR TITLE
Api enablement. Lab 5

### PR DESCRIPTION
**Pull request by tsudd**
app, connected to firestore with API

### CODE REVIEW

File: src/auth.html [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+    <link
+        href="https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,300;0,400;0,700;1,300;1,400;1,700&display=swap"
+        rel="stylesheet">
+    <script src="https://www.gstatic.com/firebasejs/ui/6.0.1/firebase-ui-auth.js"></script>
+    <link type="text/css" rel="stylesheet" href="https://www.gstatic.com/firebasejs/ui/6.0.1/firebase-ui-auth.css" />
+</head>
+
+<body>
+    <div id="sidepan" class="sidepanel">
+        <a id="closebtn" href="javascript:void(0)" class="close-btn">&times;</a>
```
Comment (katetushkan) [inline src/auth.html]: `button`.

File: src/auth.html [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+        <ul class="overlay-links">
+            <li><a class="header-link" href="index.html">Plant</a></li>
+            <li><a class="header-link" href="stats.html">Stats</a></li>
+            <li><a id="profile" class="header-link" href="auth.html" style="font-size: 30px;">Log in/Register</a></li>
```
Comment (katetushkan) [inline src/auth.html]: Inline style.

File: src/auth.html [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+                <li><a href="https://github.com/katetushkan">
+                        <svg class="header-svg header-link" width="36" height="36" viewBox="0 0 24 24"
+                            xmlns="http://www.w3.org/2000/svg" role="img">
+                            <title>Telegram</title>
+                            <path fill="currentColor" fill-rule="evenodd"
+                                d="m12 0c-6.627 0-12 5.373-12 12s5.373 12 12 12 12-5.373 12-12c0-6.627-5.373-12-12-12zm5.894 8.221-1.97 9.28c-.145.658-.537.818-1.084.508l-3-2.21-1.446 1.394c-.14.18-.357.295-.6.295-.002 0-.003 0-.005 0l.213-3.054 5.56-5.022c.24-.213-.054-.334-.373-.121l-6.869 4.326-2.96-.924c-.64-.203-.658-.64.135-.954l11.566-4.458c.538-.196 1.006.128.832.941z" />
+                        </svg></a></li>
+            </ul>
+        </nav>
+        <button class="openbtn" id="open-sidebar"><img class="overmenu" src="./assets/bar.png" width="60"
```
Comment (katetushkan) [inline src/auth.html]: Inline styles.

File: src/auth.html [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+                height="60"></button>
+    </header>
+    <main class="content">
+        <div class="wrapper">
+            <div class="container">
+                <div class="tabs">
+                    <ul>
+                        <li class="log-in-li active">Log in</li>
+                        <li class="register-li">Register</li>
```
Comment (katetushkan) [inline src/auth.html]: Links?

File: src/profile.html [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+                                d="m12 0c-6.627 0-12 5.373-12 12s5.373 12 12 12 12-5.373 12-12c0-6.627-5.373-12-12-12zm5.894 8.221-1.97 9.28c-.145.658-.537.818-1.084.508l-3-2.21-1.446 1.394c-.14.18-.357.295-.6.295-.002 0-.003 0-.005 0l.213-3.054 5.56-5.022c.24-.213-.054-.334-.373-.121l-6.869 4.326-2.96-.924c-.64-.203-.658-.64.135-.954l11.566-4.458c.538-.196 1.006.128.832.941z" />
+                        </svg></a></li>
+            </ul>
+        </nav>
+        <button class="openbtn" id="open-sidebar"><img class="overmenu" src="./assets/bar.png" width="60"
+                height="60"></button>
+    </header>
+    <main class="column info">
+        <h1 class="profile-head">Profile</h1>
+        <div>
```
Comment (katetushkan) [inline src/profile.html]: `dl` is a better tag for this data.

**Review by katetushkan — CHANGES_REQUESTED** [review]

File: src/auth.html (hunk 1 above)
Comment (tsudd) [reply src/auth.html]: button was implemented

File: src/auth.html (hunk 2 above)
Comment (tsudd) [reply src/auth.html]: inlines styles where deleted

File: src/auth.html (hunk 4 above)
Comment (tsudd) [reply src/auth.html]: didn't quite get this comment... just added <a> tags...

File: src/profile.html (hunk 5 above)
Comment (tsudd) [reply src/profile.html]: implemeted dl tag

**Review by katetushkan — APPROVED** [review]
<kbd>10</kbd>.

**Comment by tsudd** [comment]
Thanks for the review
Will provide the last PR shorlty.

**Pull request merged**